### PR TITLE
Feature/landing screen

### PR DESCRIPTION
**Pull request by hakobyaann**
A custom Button for Landing Screen. (In Models Group)

### CODE REVIEW

**Comment by nellyAslanyan** [comment]
Please add screenshot

**Comment by hakobyaann** [comment]
Screenshot 2024-10-23 at 11 02 28

**Review by iskandaryansona — DISMISSED** [review]

File: Birthday/Birthday/Screens/LandingView.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                        backgroundColor: .mainPink,
+                        action: { print("Navigation to Sign in page") },
+                        cornerRadii: [42, 0, 42, 42]
+                    )
+                    LandingButton(
+                        title: String.Button.register,
+                        textColor: .mainPink,
+                        backgroundColor: .darkRed,
+                        action: { print("Navigation to Register page") },
+                        cornerRadii: [42, 42, 42, 0]
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Screens/LandingView.swift]: Lets have this in property

**Comment by nellyAslanyan** [comment]
Check and fix indentation in affected files

File: Birthday/Birthday/Models/LandingButton.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          topTrailingRadius: cornerRadii[3]
+        )
+      )
+    }
+  }
+
+}
+
+#Preview {
+  LandingButton(title: String.Button.register,
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Models/LandingButton.swift]: title should be on new line, ) closing bracket too

File: Birthday/Birthday/Screens/ContentView.swift [hunk 3]
```diff
@@ -12,7 +12,7 @@ import BirthDayAPI
 struct ContentView: View {
   
     var body: some View {
-      TabBarView()
+        TabBarView()
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Screens/ContentView.swift]: revert  this change

File: Birthday/Birthday/Utils/Extensions/Texts.swift [hunk 4]
```diff
@@ -8,9 +8,11 @@
 import Foundation
 
 extension String {
-  
-  enum Button {
-    static var login: String { "button_login".localized }
-  }
-  
+
+    enum Button {
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Utils/Extensions/Texts.swift]: indentation here

File: Birthday/Birthday/Screens/LandingView.swift [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      EnteranceView()
+        .padding(.horizontal, 67)
+    }
+  }
+
+}
+
+struct EnteranceView: View {
+
+  var body: some View {
```
Comment (Mekhak11) [inline Birthday/Birthday/Screens/LandingView.swift]: I think no need for creating separate EntranceView and call it in LandingView, You can divide Entrance view into small parts and add those parts in extension of LandingView then use them:)

File: Birthday/Birthday/Models/LandingButton.swift [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+import SwiftUI
+
+struct LandingButton: View {
+
+  var title: String
+  var textColor: Color
+  var backgroundColor: Color
+  var action: () -> Void
+  var cornerRadii: [CGFloat]
+
```
Comment (Mekhak11) [inline Birthday/Birthday/Models/LandingButton.swift:17]: cornerRadius is better:)

**Review by nellyAslanyan — APPROVED** [review]

**Pull request merged**